src/GetUserToIdCommand.java

```diff
@@ -0,0 +1,35 @@
+import java.util.InputMismatchException;
+import java.util.Scanner;
+
+public class GetUserToIdCommand implements Command {
+    private final UsersBook usersBook;
+    private final String SEPARATOR = "____________________________________________________________________________";
+
+    public GetUserToIdCommand(UsersBook usersBook) {
+        this.usersBook = usersBook;
+    }
+
+    @Override
+    public void execute() {
+        int id;
+        while (true) {
+            System.out.println("Введите id пользователя: ");
+            Scanner scannerId = new Scanner(System.in);
+            try {
+                int inputId = scannerId.nextInt();
+                scannerId.nextLine();
+                id = inputId;
+                break;
+            } catch (InputMismatchException exception) {
+                System.out.println("Ошибка ввода, нужно вводить только числа (целочисленные)");
```

Comment:
if you find error during command execution, maybe command should stop to work ? And someone who use your command will just restart execution if needed. Think about it